src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -260,7 +260,7 @@ $headings-font-family: 'Work Sans SemiBold', sans-serif;
 $font-family-sans-serif: 'Noto Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif !default;
 
 //Border
```

Comment:
I think comments are redundant, in my opinion the variables name should say what the variable is for

Reply:
I think it's outside of the scope of this PR.